Art Institute of Chicago.js

```diff
@@ -0,0 +1,254 @@
+	"label": "Art Institute of Chicago",
+	"creator": "nikhil trivedi, Illya Moskvin",
+	"target": "^https?://(www\\.)?artic\\.edu/(artworks/|collection)",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-02-22 19:17:08"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2021 Art Institute of Chicago
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc) {
+	if (ZU.xpathText(doc, '//html[contains(@class, "p-artwork-show")]')) {
+		return 'artwork';
+	}
+	else if (ZU.xpathText(doc, '//html[contains(@class, "p-collection-index")]')) {
+		return 'multiple';
+	}
+	return false;
+}
+
+function doWeb(doc) {
+	if (detectWeb(doc) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
+			}
+			var articles = [];
+			for (var i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+			return true;
+		});
+	} else {
+		scrape(doc);
+	}
+}
+
+function scrape(doc) {
+	var item = new Zotero.Item('artwork');
+	item.title = ZU.xpathText(doc, '//h1');
+
+	var artists = ZU.xpath(doc, '//meta[contains(@name, "citation_author")]');
+	for (var i = 0; i < artists.length; i++) {
+		var cleaned = artists[i].content.replace(/\(.*\)$/, '').trim();
+		if (cleaned.split(' ').length > 2) {
+			item.creators.push({ lastName: cleaned, creatorType: 'artist', fieldMode: 1 });
+		}
+		else {
+			item.creators.push(ZU.cleanAuthor(cleaned, 'artist'));
+		}
+	}
+
+	item.date = ZU
+		.xpath(doc, '//dl[@id="dl-artwork-details"]/dd[@itemprop="dateCreated"]/*/a')
+		.map(function (date) {
+			return date.textContent;
+		})
+		.join(', ');
+
+	item.artworkMedium = ZU.xpathText(doc, '//dl[@id="dl-artwork-details"]/dd[@itemprop="material"]');
+	item.artworkSize = ZU.xpathText(doc, '//dl[@id="dl-artwork-details"]/dd[@itemprop="size"]');
+	item.callNumber = ZU.xpathText(doc, '//dl[@id="dl-artwork-details"]/dd[@itemprop="identifier"]');
+
+	item.url = ZU.xpathText(doc, '//link[@rel="canonical"]/@href');
+
+	item.complete();
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = ZU.xpath(doc, '//ul[@id="artworksList"]/li/a');
+	for (var i=0; i<rows.length; i++) {
```

Comment:
Add spacing here to make the linter happy! (I think this is the old template's fault.)

Reply:
I think I addressed the suggestions you raised. Let me know if you want me to square the commit, rebase, or make any other changes!

Reply:
Great, thanks! It looks like all that's left is to update the tests in Scaffold. It'd be helpful if you could rebase, too. I'll squash when I merge everything in.

Reply:
CI will probably keep failing, because it doesn't deal well with the branch point being too far in the past, but that's fine as long as tests pass locally.